src/pages/Label/Label.tsx

```diff
@@ -24,7 +25,24 @@ export const Label = () => {
         </section>
       ) : (
         labelNotes.map((note) => {
-          return <p>{note.noteText}</p>;
+          return (
+            <article
+              className={`p-2 my-5 mx-5 border-2 border-black rounded-lg max-w-3xl mx-auto whitespace-pre-wrap break-words bg-${note.bgColor}`}
+            >
+              <div>
+                <h2 className="text-xl font-semibold text-xl mb-3">
+                  {note.title}
+                </h2>
+                <p>{note.noteText}</p>
+              </div>
+
+              <div className="my-2 flex">
+                {note.labels.map((label) => (
+                  <LabelTag currentLabel={label} noteId={note.id} />
```

Comment:
suggestion: it seems you forgot to write "key" attribute.

Reply:
Noted, changes will be made in future PRs